tests/010-test-use-devs-to-create-docker-root-volume.sh

```diff
@@ -19,6 +19,7 @@ test_docker_root_volume() {
 DEVS="$devs"
 VG=$vg_name
 DOCKER_ROOT_VOLUME=yes
+DOCKER_ROOT_VOLUME_SIZE=40%FREE
```

Comment:
We are removing this from defaults, since we don't want people using it any longer.